pkg/vsphere/session/session.go

```diff
@@ -319,6 +319,28 @@ func (s *Session) Folders(ctx context.Context) *object.DatacenterFolders {
 	return s.folders
 }
 
+// QueryOption looks up the option's values and returns the first one from the
+// ones received on success.
+func (s *Session) QueryOption(ctx context.Context, option string) (string, error) {
+	client := s.Vim25()
+	if client == nil {
```

Comment:
Originally added this as a fail-safe, but I traced the validator/session calls and looks like the client will be non-nil here since errors from `session.Connect()` are not ignored. I'll remove this check.